src/components/Trigger/Trigger.tsx

```diff
@@ -14,7 +14,7 @@ export enum TriggerSize {
     Large = 'Large',
 }
 
-export enum InputEmphasis {
+export enum TriggerEmphasis {
```

Comment:
This will add a breaking change, if it is being used outside of Fondue.
I checked Clarify and couldn't find any usage of it.
Also checked that it was created by @Walkeezy [here](https://github.com/Frontify/fondue/pull/1076/files).
Idk if he is using outside of Fondue, so it would be nice to double check if these changes will not break anything before merging them.

Reply:
Going to use it in Frontify/guideline-blocks. It's fine to rename it from my side. :-)

Reply:
Nice, thanks. Let's do `TriggerEmphasis` for consistency then 👍